src/internal/components/cartesian-chart/label-utils.ts

```diff
@@ -80,3 +80,12 @@ function hasIntersection(a: FormattedTick, b: FormattedTick) {
   const rightEdge = right.position - right.space / 2;
   return leftEdge > rightEdge;
 }
+
+/* istanbul ignore next */
```

Comment:
I'd add a comment to justify why part of the code can be uncovered

Reply:
I prefer ignoring the lines that are not going to be covered due to the limitations like jsdom support or, say, invariant violation exceptions. I am happy to have less than 100% coverage but I think it is great when the missing lines shown in the report represent the missing opportunity.

Reply:
Even in this case, you can still narrow it down to a specific condition instead of marking the whole function

Reply:
Any news on this thread?

If you wanted to ignore only JSDOM branch, it should be this way

```js
function getSVGTextSize(element: null | SVGTextElement): undefined | { width: number; height: number } {
  /* istanbul ignore if  */
  if (element && element.getBBox) {
    return element.getBBox();
  }
  return undefined;
}

```

Reply:
Thanks, I am familiar with the syntax. The problem is that it will only ignore the line inside of the IF not ignoring the function itself and the two remaining lines. That is enough to make the diff coverage check unhappy. At the same time, the entire function is meant to be mocked. I can add a test for the function alone to make it covered but that does not seem that necesary.

Reply:
So, you created a problem for yourself by choosing too aggressive mocking strategy: https://github.com/cloudscape-design/components/pull/1694#discussion_r1373437193

Ignoring this is not fine, it needs to be solved properly. I recommend revisit the mocks, they are hiding too much

Reply:
This function returns the BBox of an element, what do we win if get it covered?

Reply:
What do we win from marking it as ignored?

Reply:
The last statement `return undefined` is not necessary. Would removing that line help?

Or alternatively, write it in a single line:

```
return element && element.getBBox ? element.getBBox() : undefined;
```